### PR TITLE
Add field to admin submission section

### PR DESCRIPTION
**Pull request by devkonrad**
Add submission fields and changes the development platform dispĺay contidion for datagrid on admin interface

Changes:
modified: app/data_grids/submissions_grid.rb

Refs: https://github.com/Iridescent-CM/technovation-app/issues/3196

### CODE REVIEW

**Review by viviancan — COMMENTED** [review]
Lgtm! Pulled down and tested locally. Will wait for Shaun's review as well!

File: app/data_grids/submissions_grid.rb [hunk 1]
```diff
@@ -80,6 +82,13 @@ class SubmissionsGrid
     team.city
   end
 
+  column :development_platform do
+    if development_platform_text.present?
+      platform = development_platform_text 
+      platform.downcase.starts_with("mobile") ? platform : ""
```
Comment (viviancan) [inline app/data_grids/submissions_grid.rb:88]: This errored out for me locally when trying to list the development platform. 
Is the `starts_with` missing a `?` 

`starts_with?`

**Review by viviancan — COMMENTED** [review]
I had an error when selecting the development_platform option in the admin portal! Please verify

File: app/data_grids/submissions_grid.rb (hunk 1 above)
Comment (devkonrad) [reply app/data_grids/submissions_grid.rb:88]: Sorry @viviancan for this. You are correct, is missed out the interrogation point.

**Review by viviancan — APPROVED** [review]
LGTM! Thanks for tackling this one! Please merge into QA 👍

**Comment by devkonrad** [comment]
> LGTM! Thanks for tackling this one! Please merge into QA +1

Thanks, @viviancan! Merged into QA! :)

**Pull request merged**